rfid_tracker/app.py

```diff
@@ -7,10 +7,13 @@
 import typer
 from mfrc522 import SimpleMFRC522
 from models import Action, Base, Member
+from slack_webhook import Slack
 
 MAX_SPOTS = 19
+WEBHOOK_URL = 'https://hooks.slack.com/services/xxx/yyy'
```

Comment:
I think this should be removed, and rather... (see next comment)

Reply:
Done